manifests/base/gateway-controller/gateway-controller-deployment.yaml

```diff
@@ -1,25 +0,0 @@
-apiVersion: apps/v1
```

Comment:
I think we should keep the gateway-controller running for some time, it is used to clean up old generated gateway deployment and service. Or you have a better way for upgrade?

Reply:
I have the gateway controller removed from normal installation/manifests because for new users, we want them to just install the latest components. I'll add a gateway-controller manifest in the `migration path` documentation.

Reply:
👍